docs/root/faq/api/control_plane_version_support.rst

```diff
@@ -30,6 +30,11 @@ typical rollout sequence might look like:
 4. Support for v2 is removed in the management server. The management server moves to v3 exclusively
    internally and can support newer fields.
 
+5. Client can send discovery request with v2 resource type url and process discovery response with 
+   v3 resource type url. Client can also send discovery request with v3 resource type url and process 
+   discovery response with v2 resource type url. The upgrade and downgrade of type url is disabled 
+   by default and protected by a runtime guard *envoy.reloadable_features.enable_type_url_downgrade_and_upgrade*.
```

Comment:
Still need to add an entry in the version history, see https://www.envoyproxy.io/docs/envoy/latest/version_history/current

Reply:
Thanks, will do.